docs/configuring.md

```diff
@@ -830,6 +830,26 @@ Suppress status output.
 
     Not supported in `copier.yml`.
 
+### `secret_questions`
+
+-   Format: `List[str]`
+-   CLI flags: N/A
+-   Default value: N/A
```

Comment:
```suggestion
-   Default value: `[]`
```

Reply:
Sounds good.

But then I think [`migrations`](https://copier.readthedocs.io/en/latest/configuring/#migrations), [`skip_if_exists`](https://copier.readthedocs.io/en/latest/configuring/#skip_if_exists), and [`tasks`](https://copier.readthedocs.io/en/latest/configuring/#tasks) also need an empty list as the default value. By the way, [`jinja_extensions`](https://copier.readthedocs.io/en/latest/configuring/#jinja_extensions) already has an empty list as the default value.

I can create a separate PR for the other corrections, if you agree that's how it should be.

Reply:
WDYT about the other settings, @yajo?

Reply:
Yes, we should correct them too 👍